### DISCUSSION TITLE
Ignore sliderstartcircle on skinned beatmaps

### DISCUSSION THREAD

**Discussion by JamieWright99** [ppy/osu#30927] [Ideas] [answered]
When I use a skin that has sliderstartcircle assets it creates a mismatch when playing skinned beatmaps. I think it would be nice if slidercircle files are ignored here and replaced by the beatmap hitcircle, unless the mapper has included their own slidercircle files. 
![image](https://github.com/user-attachments/assets/06f369b3-4397-4740-bcbc-5e940d186131)

**Comment by peppy** [comment]
I'm not sure what you're showing here sorry.

Please include your skin and describe what you're seeing/expecting again to help me understand.

**Reply by JamieWright99** [reply]
Sorry about that, here's a skin showing what I mean ( [Cookiezi 2018 (Seouless).zip](https://github.com/user-attachments/files/17987908/Cookiezi.2018.Seouless.zip) ) and here's an example map https://osu.ppy.sh/beatmapsets/1626624#osu/3322444

The beatmap comes with skin elements for hitcircles but not any for slider circles so it will use the one from the user skin. I'm saying this looks weird most of the time and pulls away from the mappers intentions.  I hope this is clearer \o/

**Reply by peppy** [reply]
Right, thanks.

**Reply by peppy** [reply, marked as answer]
https://github.com/ppy/osu/pull/31052
